src/domain/wallets/tx-history.ts

```diff
@@ -118,7 +119,7 @@ export const fromLedger = (
             settlementVia: {
               type: SettlementMethod.IntraLedger,
               counterPartyWalletId: recipientWalletId as WalletId,
-              counterPartyUsername: username || null,
+              counterPartyUsername: username,
```

Comment:
are we sure we dont have null values in db?

Reply:
did you test this at graphql level?

Reply:
How do you mean? We're casting falsy values to `undefined` as they're being read out of the database either way [here](https://github.com/GaloyMoney/galoy/pull/1077/files#diff-463de0ec8400f9be4647cb42dee40f257b5055c84cbe9328daa260760f4a1904R422)